### ISSUE TITLE
Check-checker getting hung up on spread operator

### ISSUE THREAD

**Issue by DevanB** [East5th/check-checker#8]
I just noticed a server side console log coming from check-checker…

```
While linting files with east5th:check-checker for the app (Server):
   server/publications.js:378:37: Unexpected token …
```

Looks like it’s getting hung up on the spread operator:

`self.added("memberCrfs", id, { …fields, …member });`

**Comment by pcorey** [comment]
Good catch. I'll look into it.

**Comment by pcorey** [comment]
Hey @DevanB,

It looks like you're using a [unicode ellipsis](http://www.charbase.com/2026-unicode-horizontal-ellipsis) (`…`) there instead of the actual spread operator (`...`). A lot of places like Google Docs will automatically translate `...` to `…` - maybe that's the culprit here?

Check-checker (and the es6 compiler in general) complains about the second line in this method, but not the first.

``` javascript
"spread": function(hello) {
  console.log(...arguments);
  console.log(…arguments);
}
```

**Comment by DevanB** [comment]
Odd, I think Chrome/Safari replaced that with the unicode ellipsis. I just verified we are using three periods in a row. Let me try again:

`self.added("memberCrfs", id, { ...fields, ...member });`

It is still giving the same output:

```
 While linting files with east5th:check-checker for the app (Server):
   server/publications.js:378:38: Unexpected token ...
```

EDIT: Should have continued reading. Didn't read where you said it wasn't complaining about a correct spread operator. Anything I can do somehow to help since I'm still seeing the incorrect output?

**Comment by pcorey** [comment]
Yeah, my mistake. It was a bug. Try upgrading to `@0.2.0` - it should be fixed.

Let me know if you have any more issues!
